lisa/schema.py

```diff
@@ -1487,6 +1487,7 @@ class TestCase(BaseTestCaseFilter):
             field_function=fields.Int, validate=validate.Range(min=0)
         ),
     )
+    timeout: int = 3600
```

Comment:
It needs to be optional, so if some test case is shorter timeout, it should be respected.

Reply:
made it optional